media-libs/gmmlib/gmmlib-21.3.5.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Intel Graphics Memory Management Library"
+HOMEPAGE="https://github.com/intel/gmmlib"
+SRC_URI="https://github.com/intel/gmmlib/archive/intel-${P}.tar.gz"
+S="${WORKDIR}/${PN}-intel-${P}"
+
+KEYWORDS="~amd64"
+LICENSE="MIT"
+SLOT="0/0"
```

Comment:
I am not sure, but would it make sense, to use ```0/11``` as slot, since 21.3.5 has ```libigdgmm.so.11```, while >= 22.0.0 has ```libigdgmm.so.12``` and with 22.0.0 and 9999 you are using already ```0/12```?

Reply:
Other versions of `gmmlib` have "default" subslot 0, this version just reuse existing number for subslot to avoid creation of the new revisions for old `gmmlib` ebuilds.
There are no strict rules to have same number for subslot and soname, however it makes sense to follow this naming conventions for next versions with non-default subslot.

Reply:
That's fine for me. Just wanted to check, that this is, what you want.